sda-commons-starter/README.md

```diff
@@ -0,0 +1,128 @@
+# SDA Commons Starter
+
+[![javadoc](https://javadoc.io/badge2/org.sdase.commons/sda-commons-starter/javadoc.svg)](https://javadoc.io/doc/org.sdase.commons/sda-commons-starter)
+
+> #### ⚠️ Experimental ⚠
+>
+> Please be aware that this API is in an early stage and might change in the future.
+>
+
+The module `sda-commons-starter` provides all basics required to build a service for the SDA Platform with
+Dropwizard.
+
+Apps built with the [`SdaPlatformBundle`](./src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java)
+automatically contain
+
+- [Support for environment variables in configuration files and default console appender configuration](../sda-commons-server-dropwizard/README.md)
+- [Trace Token support](../sda-commons-server-trace/README.md)
+- [a tolerant `ObjectMapper`, HAL support and a field filter](../sda-commons-server-jackson/README.md)
+- [Security checks on startup](../sda-commons-server-security/README.md)
+- [Authentication support](../sda-commons-server-auth/README.md)
+- [Prometheus metrics](../sda-commons-server-prometheus/README.md)
+- [OpenApi documentation](../sda-commons-server-openapi/README.md)
+- [Open Tracing](../sda-commons-server-opentracing/README.md) and [Jaeger](../sda-commons-server-jaeger/README.md)
+
+They may be configured easily to
+
+- [allow cross origin resource sharing](../sda-commons-server-cors/README.md)
+- [use the Open Policy Agent for authorization](../sda-commons-server-auth/README.md)
+
+Using the [`SdaPlatformBundle`](./src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java) is the easiest
+and fastest way to create a service for the SDA Platform.
+
+To bootstrap a Dropwizard application for the SDA Platform only the 
+[`SdaPlatformBundle`](./src/main/java/org/sdase/commons/starter/SdaPlatformBundle.java) has to be added. The 
+API should be documented with Swagger annotations: 
+
+```java
+import io.dropwizard.Application;
+import io.dropwizard.setup.Bootstrap;
+import io.dropwizard.setup.Environment;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import SdaPlatformBundle;
+import SdaPlatformConfiguration;
+
+@OpenAPIDefinition(info = @Info(
+    title = "My First App",
+    description =
+        "The description of my first application",
+    version = "1.0.0",
+    contact =
+    @Contact(
+        name = "John Doe",
+        email = "info@example.com",
+        url = "j.doe@example.com"),
+    license = @License(name = "Sample License")))
```

Comment:
We often store all rest-interfaces in a separate package that should also contain this definition. It is not always the best idea to register the application as a starting point. Sometimes the teams use generated client-code which also includes openapi-annotations. And we don't want these client interfaces be part of the OpenAPI file of the service.

But nothing we need to fix now.